source/features/rgh-linkify-features.tsx

```diff
@@ -10,8 +10,16 @@ import onConversationHeaderUpdate from '../github-events/on-conversation-header-
 
 function linkifyFeature(codeElement: HTMLElement): void {
 	const id = getNewFeatureName(codeElement.textContent!) as FeatureID;
-	if (features.list.includes(id) && !codeElement.closest('a')) {
-		wrap(codeElement, <a href={`/refined-github/refined-github/blob/main/source/features/${id}.tsx`}/>);
+	if (features.list.includes(id)) {
+		const href = `/refined-github/refined-github/blob/main/source/features/${id}.tsx`;
+
+		const possibleLink = codeElement.firstElementChild ?? codeElement;
+		if (possibleLink instanceof HTMLAnchorElement) {
+			possibleLink.href = href;
+			possibleLink.classList.add('color-fg-accent');
```

Comment:
If the blueish looks too much, we can use `.Link--primary` instead which only highlights in blue when hovering.